packages/client/src/components/Footer/Footer.component.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Footer.style.css';
+import { Link } from "react-router-dom";
+
+export const Footer = () => {
+  return (
+    <footer className="footer-container">
+      <div className="footer-content-section">
+        <article>
```

Comment:
I still disagree with using `article` here, it means something specific, more like a "newspaper article". I would still advice using a div here since I don't see why this content needs further semantic meaning than the `footer` tag already gives.

Reply:
Thank you for your quick response, we fixed it all .